tools/docker/firecracker-runtime.json

```diff
@@ -7,5 +7,10 @@
   "cpu_template": "T2",
   "log_fifo": "/tmp/fc-logs.fifo",
   "log_level": "Debug",
-  "metrics_fifo": "/tmp/fc-metrics.fifo"
+  "debug": true,
+  "metrics_fifo": "/tmp/fc-metrics.fifo",
+  "jailer": {
+    "uid": 123,
```

Comment:
Can we use the names instead of the IDs? According to [runtime-spec](https://github.com/opencontainers/runtime-spec/blob/7a49e344a7639b84278dda9ad0671298d223c661/config.md#posix-platform-user)

> Note: symbolic name for uid and gid, such as uname and gname respectively, are left to upper levels to derive (i.e. /etc/passwd parsing, NSS, etc)

Except for root (uid=0), it is hard to guess the meaning of the IDs.

Reply:
since runc uses `uid` and `gid`, I'd rather stick with that to be consistent